### PR TITLE
[WIP] Openshift deployment changes

### LINKED ISSUE #106: shared pvc name
## Expected Behavior

Components CRs other than koji-hub should use the koji-hub pvc name as their shared mnt pvc name. 

## Actual Behavior

Some components such as koji-builder are creating their own "shared pvc" volume instead of using the existing one by default.

## Fix

Ensure that all CRs in `mbox-operator/deploy` folder are using the same pvc name used in koji-hub cr.

### PR DESCRIPTION
**Pull request by Zlopez**
Signed-off-by: Michal Konečný <mkonecny@redhat.com>

Fixes #106

## Proposed Changes

* Fix for postgres deployment in openshift
* Typo fix in makefile
* Share PVC use for koji

## Verification Steps

* Tests must pass
* Deployment in staging openshift is completed

### CODE REVIEW

File: docs/source/user-guide/koji-hub.rst [hunk 1]
```diff
@@ -222,6 +230,34 @@ Available choices:
 
 * nginx
 
+httpd_pvc_name
+--------------
+
+Name of the PersistentVolumeClaim for httpd server koji-hub will use.
+
+If provided PVC doesn't exists, it creates it's own.
```
Comment (odra) [inline docs/source/user-guide/koji-hub.rst:238]: `it's` -> `its`

File: docs/source/user-guide/koji-hub.rst [hunk 2]
```diff
@@ -222,6 +230,34 @@ Available choices:
...
+Size of the PersistentVolumeClaim for httpd server koji-hub will create.
+
+If httpd_pvc_name exists, this value is ignored.
+
+mnt_pvc_name
+------------
+
+Name of the PersistentVolumeClaim koji-hub will use.
+
+If provided PVC doesn't exists, it creates it's own.
```
Comment (odra) [inline docs/source/user-guide/koji-hub.rst:252]: same as above

**Comment by odra** [comment]
looks good, just two comments.

File: docs/source/user-guide/koji-hub.rst (hunk 1 above)
Comment (Zlopez) [reply docs/source/user-guide/koji-hub.rst:238]: I think this should be ok, "it's" means that it belongs to it. Same as Sam's or Leon's. But I will rather check.

Comment (Zlopez) [reply docs/source/user-guide/koji-hub.rst:238]: According to [this](https://www.dictionary.com/e/its-vs-its/), you are right. English grammar is sometimes strange.

**Comment by odra** [comment]
I think the tests are failing because of the pvc name in our asserts playbooks, those are hardcoded there.

**Comment by Zlopez** [comment]
There was one fail because of the size of one PVC, but I need to update tests because of the name change.

**Pull request merged**